docs/developers/NewToGluten.md

```diff
@@ -78,18 +76,18 @@ If you need to debug the tests in <gluten>/gluten-ut, You need to compile java c
 
 # Java/scala code development with Intellij
 
-## Linux intellij local debug
+## Linux IntelliJ local debug
 
-Install the linux intellij version, and debug code locally.
+Install the Linux IntelliJ version, and debug code locally.
 
 - Ask your linux maintainer to install the desktop, and then restart the server.
 - If you use Moba-XTerm to connect linux server, you don't need to install x11 server, If not (e.g. putty), please follow this guide:
 [X11 Forwarding: Setup Instructions for Linux and Mac](https://www.businessnewsdaily.com/11035-how-to-use-x11-forwarding.html)
 
-- Download [intellij linux community version](https://www.jetbrains.com/idea/download/?fromIDE=#section=linux) to linux server
+- Download [IntelliJ Linux community version](https://www.jetbrains.com/idea/download/?fromIDE=#section=linux) to Linux server
 - Start Idea, `bash <idea_dir>/idea.sh`
 
-Notes: Sometimes, your desktop may stop accidently, left idea running.
+Notes: Sometimes, your desktop may stop accidentally, left idea running, you need to kill its process explicitly:
```

Comment:
I think this note is too trivial and it rarely happens. Let's remove this line and the following relevant content.